pkg/postage/batchservice/batchservice.go

```diff
@@ -146,8 +147,10 @@ func (svc *batchService) TopUp(id []byte, normalisedBalance *big.Int, txHash []b
 		return fmt.Errorf("update: %w", err)
 	}
 
+	topUpAmount := big.NewInt(normalisedBalance.Int64() / int64(math.Pow(2, float64(b.Depth))))
```

Comment:
please use big int division, something like 
`topUpAmount := new(big.Int).Div(normalisedBalance, 1 << b.depth)`

Reply:
This is called normalisedBalance but it is the topup amount per chunk. If you divide it by no. of chunks what would this value be?

Reply:
@aloknerurkar is right, @swarmHaseeb you missed the change we discussed which was passing `TopUpAmount` to `TopUp` from inside postage/listener.go line 140

Reply:
[it should be new(big.Int).Div(topUpAmount, 1 << b.depth)](https://github.com/ethersphere/storage-incentives/blob/master/src/PostageStamp.sol#L151)